packages/universal-components/src/Button/Button.stories.js

```diff
@@ -8,6 +8,7 @@ import {
   select,
   text,
   number,
+  object,
```

Comment:
@KarinaDavtyan Just remove this one and we can merge it 🚀

Reply:
Done